index.js

```diff
@@ -27,7 +27,7 @@ const PAGE_CREATE_DIALOG = {
       section: null,
       templates: [],
       template: '',
-      page: {},
+      page: this.emptyForm(),
```

Comment:
I'd like to stick with:

```suggestion
      page: {},
```

Line 74 and follow are dedicated to populate the `page` object according to the fields. But [`emptyForm()`](https://github.com/getkirby/kirby/blob/7525553b8d9976d6ed08b702f62fc3d368116777/panel/src/components/Dialogs/PageCreateDialog.vue#L66-L72) populates the properties `title`, `slug` and `template` only.

Reply:
I just reverted that change.